docs/api-docs/storefront/scripts-overview.mdx

```diff
@@ -69,7 +69,7 @@ For more on checking which theme engine a storefront is using, see this article'
 
 ## Managing scripts using single-click app callbacks
 
-If you're using an [app-level API account](/api-docs/getting-started/api-accounts#app-level-api-accounts), we recommend inserting your scripts immediately following completion of the [auth callback](/api-docs/apps/guide/auth). The app can use the [uninstall callback](/api-docs/apps/guide/callbacks) to remove its scripts when a merchant uninstalls it, and the [remove_user callback](/api-docs/apps/guide/callbacks) to remove any scripts related to a particular user when they lose access privileges.
+If you're using an [app-level API account](/api-docs/getting-started/api-accounts#app-level-api-accounts), we recommend inserting your scripts immediately following completion of the [auth callback](/api-docs/apps/guide/auth). The app can use the [uninstall callback](/docs/integrations/apps/guide/callbacks) to remove its scripts when a merchant uninstalls it, and the [remove_user callback](/api-docs/apps/guide/callbacks) to remove any scripts related to a particular user when they lose access privileges.
```

Comment:
these 2 `/api-docs/apps/guide/auth` & `/api-docs/apps/guide/callbacks` I am getting different links to ie `/docs/integrations/apps/guide/auth` & `/docs/integrations/apps/guide/callbacks`

Reply:
fixed i believe